python/heterocl/report.py

```diff
@@ -61,9 +406,19 @@ def parse_xml(path, print_flag=False):
     splitline = "+" + endash[1] + "+" + endash[2] + "+"
     tablestr.insert(5, splitline)
     table = '\n'.join(tablestr)
+
+    # Latency information extraction
+    clock_unit = profile["PerformanceEstimates"]["SummaryOfOverallLatency"]["unit"]
+    summary = profile["PerformanceEstimates"]["SummaryOfLoopLatency"]
+
+    info_table = Displayer(clock_unit)
+    info_table.scan_range(summary)
+    info_table.get_loops(summary)
+    info_table.init_data(summary)
```

Comment:
We should merge these three into a single method.

Reply:
I could not find a way to combine them into one function since they all depend on each other to complete first (need to be executed sequentially to initialize member fields appropriately).